fortuna/src/MarketPlace.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+  
+  //This is the MarketPlace Display
+  class MarketPlaceDisplay extends React.Component {
```

Comment:
The class name needs to match the file name so we don't get super confused when we are importing stuff.